docs/tutorials/data-access.txt

```diff
@@ -7,8 +7,9 @@ Nearly all experiments require training and validation
 data. In Determined, data access depends on the cluster infrastructure and the data storage.
 
 At the end of this tutorial, the reader should know:
```

Comment:
non-blocking: `the reader will know`, maybe even make it `the reader will know best practices for`